tests/application/static_site/test_views.py

```diff
@@ -440,6 +440,43 @@ def test_measure_page_social_sharing(app, test_app_client, logged_in_rdu_user):
         assert twitter_sharing_meta[0].get("content") == value
 
 
+def test_measure_page_social_sharing_using_description(app, test_app_client, logged_in_rdu_user):
```

Comment:
We could do this as a parameterized test, as this is basically a fairly long copy/paste of the above test. It's not the end of the world to have it duplicated, but I think using the same test makes it clearer that these are testing the same thing with just minor variations. Also probably more maintainable.

Parameterized tests are kinda confusing at first glance though. Here's roughly how they work.

Add the decorator `@pytest.mark.parametrize("var1_name, var2_name", ((var1_val1, var2_val1), (var1_val2, var2_val2)))`. Then add var1 and var2 to the signature of the decorated method. Then use those in the body of the test to set up and test your variations.

For example in this test we might a parameterization that looks like this:
```@pytest.mark.parametrize(
    "summary, description, expected_social_description",
    (
        ("Unemployment Summary\n * This is a summary bullet", None, "* This is a summary bullet"),
        ("Unemployment Summary\n * This is a summary bullet", "This is the short description", "This is the short description"),
    )
)```

Reply:
Yeah, this is probably a good call here, given the repeated code.

The annoying thing I find about parameterised tests though is that the output is less readable as you lose the ability to describe each of the scenarios using the method name. But that's probably not a huge deal though, especially here. Will change.

Reply:
I know what you mean. You should be able to describe roughly what's being tested, and if that's combined with good explanation text on asserts then it's usually OK.